### DISCUSSION TITLE
custom height and width of gcanvas

### DISCUSSION THREAD

**Discussion by hunteronoffical** [flyskywhy/react-native-gcanvas#63] [Q&A]
hi !

i have simple code. if i fixed height of canvas, cavas display but  this code is not display. please help me 

import React, { useState, useContext, useEffect } from 'react';
import { GCanvasView, GImage } from '@flyskywhy/react-native-gcanvas';
const Test = (height=300) => {
    const [canvasOff, setcanvasOff] = useState(null);
    useEffect(() => {
        console.log("asdfasdfg")
        if (canvasOff) {
            canvasOff.width = 1000;
            canvasOff.height = heightx;           
            const ctx = canvasOff.getContext('2d');            
            ctx.beginPath();
            //rect
            ctx.fillStyle = 'red';
            ctx.fillRect(0, 0, 50, 50);
            //rect
            ctx.fillStyle = 'green';
            ctx.fillRect(50, 50, 50, 50);
            ctx.fill();
            ctx.render()
        }
    }, []
    )   
    return (
        <>
            <GCanvasView 
                onCanvasResize={onCanvasResize} onCanvasCreate={(cavans) => { setcanvasOff(cavans),console.log(heightx) }} height={height} style={{  width: '100%', }} />
        </>
    );
}
export default Test;

**Comment by flyskywhy** [comment]
Maybe you need `flex: 1` in your style, maybe other thing, you can try, to try remove some code from the running example code in `READEM.md`, and turns the example code into yours one by one, then you will find which code cause the problem :stuck_out_tongue_closed_eyes:
